persist/encoding/msgpack/decoder.go

```diff
@@ -141,13 +153,45 @@ func (dec *decoder) decodeIndexInfo() schema.IndexInfo {
 	indexInfo.Start = dec.decodeVarint()
 	indexInfo.BlockSize = dec.decodeVarint()
 	indexInfo.Entries = dec.decodeVarint()
+	indexInfo.MajorVersion = dec.decodeVarint()
+	indexInfo.Summaries = dec.decodeIndexSummariesInfo()
+	indexInfo.BloomFilter = dec.decodeIndexBloomFilterInfo()
```

Comment:
index files are per shard / block so we have a bloomfilter per shard/block right?

Reply:
Correct yup.